trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -146,13 +147,16 @@ def spin(self,conditions):
         """         
            
         #Unpack    
-        B       = self.number_blades
+        B       = self.number_of_blades
+        E       = self.number_of_engines
```

Comment:
I know you're trying to output a set of results for later. But a number of engines for a single propeller makes no sense.

Reply:
Yes, I use this for acoustics. I broke  up a large PR (this was part of the noise PR this is coming shortly)